kube-runtime/src/controller/mod.rs

```diff
@@ -749,7 +759,7 @@ where
             error_policy,
             context,
             self.reader,
-            self.trigger_selector
+            StreamBackoff::new(self.trigger_selector, self.trigger_backoff)
```

Comment:
This applies a unified backoff timer to all triggers, do we want to maintain separate timers per trigger instead?

Reply:
Given that we are basing our default_backoff on the client-go use case (backing off because we believe the api-server is unhealthy) i think it's reasonable to keep a single, shared trigger.

Reply:
Fair enough. My counter would be that there are some cases where the error only concerns one of the watches (CRD not found, permission denied, and so on), in which case keeping the other streams going would still be a good thing.